Index: src/main/java/com/application/hotspotapplication/requests/hotspots/LocationService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.application.hotspotapplication.requests.hotspots;\r\n\r\nimport com.application.hotspotapplication.exceptions.ApiRequestException;\r\nimport java.util.List;\r\nimport java.util.stream.Collectors;\r\nimport org.springframework.beans.factory.annotation.Autowired;\r\nimport org.springframework.http.HttpStatus;\r\nimport org.springframework.stereotype.Service;\r\n\r\n@Service\r\npublic class LocationService {\r\n\r\n  @Autowired\r\n  private LocationDAO dao;\r\n\r\n\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/application/hotspotapplication/requests/hotspots/LocationService.java b/src/main/java/com/application/hotspotapplication/requests/hotspots/LocationService.java
--- a/src/main/java/com/application/hotspotapplication/requests/hotspots/LocationService.java	(revision c93591c477e57b5e6cb46e17cdee45b61ef07888)
+++ b/src/main/java/com/application/hotspotapplication/requests/hotspots/LocationService.java	(date 1618922421711)
@@ -1,10 +1,6 @@
 package com.application.hotspotapplication.requests.hotspots;
 
-import com.application.hotspotapplication.exceptions.ApiRequestException;
-import java.util.List;
-import java.util.stream.Collectors;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 
 @Service
